docs/CustomTLS.md

```diff
@@ -0,0 +1,125 @@
+# Use custom TLS certificate for Kafka
+
+TODO: Move this doc to official dcos docs
```

Comment:
Did you mean to remove this `TODO` here?

Reply:
Removed.